adapters/adapter.go

```diff
@@ -9,32 +9,42 @@ import (
 	"time"
 )
 
+// Adapters connect prebid-server to a demand partner. Their primary purpose is to produce bids
+// in response to Auction requests.
 type Adapter interface {
+	// A name which uniquely identifies this adapter.
+	// This must not overlap with any other adapters in prebid-server.
 	Name() string
 	FamilyName() string
```

Comment:
FamilyName is basically the cookie space that the adapter uses - for instance multiple adapters share the adnxs.com cookie space